codespan-reporting/src/term/views.rs

```diff
@@ -286,7 +313,11 @@ where
                 )?;
             }
 
-            let mut lines = labeled_file.lines.into_iter().peekable();
+            let mut lines = labeled_file
+                .lines
+                .iter()
+                .filter(|(_, line)| line.must_render)
+                .peekable();
             let current_labels = Vec::new();
```

Comment:
Ahh! I think I intended `current_labels` to be a stack originally. I should have put a `TODO` on this. Perhaps it can be removed now? The other usages can now be replaced with `&[]`, which is a bit more illustrative, I think. I can do this in another PR if you like, I know this has taken a while to review, so I'm happy to merge now if you like to get it off your hands.